source/robot_model/include/robot_model/Model.hpp

```diff
@@ -1,18 +1,18 @@
 #pragma once
 
-#include <iostream>
-#include <string>
-#include <vector>
 #include <OsqpEigen/OsqpEigen.h>
-#include <pinocchio/parsers/urdf.hpp>
-#include <pinocchio/multibody/data.hpp>
+#include <iostream>
```

Comment:
I think <iostream> might only be needed in the .cpp file right? (for the urdf loading)